python/sglang/srt/models/deepseek_v2.py

```diff
@@ -456,10 +476,15 @@ def forward_normal_dual_stream(
         with torch.cuda.stream(self.alt_stream):
             # router_logits: (num_tokens, n_experts)
             router_logits = self.gate(hidden_states)
-            topk_output = self.topk(hidden_states, router_logits)
-            final_hidden_states = self.experts(
-                hidden_states=hidden_states, topk_output=topk_output
-            )
+            if self.topk is not None:
+                topk_output = self.topk(hidden_states, router_logits)
+                final_hidden_states = self.experts(
+                    hidden_states=hidden_states, topk_output=topk_output
+                )
+            else:
+                final_hidden_states = self.experts(
+                    hidden_states=hidden_states, router_logits=router_logits
+                )
```

Comment:
nit: is it possible we somehow make the two branches into one unified one

Reply:
Done. PTAL.